lib/add_magic_comment.rb

```diff
@@ -1,47 +1,37 @@
 # frozen_string_literal: true
 
-# A simple library to prepend magic comments to multiple ".rb" files
+# A simple library to prepend magic comments to all Ruby files in a given folder
 
 module AddMagicComment
   MAGIC_COMMENT_PREFIX  = "frozen_string_literal"
   MAGIC_COMMENT_PATTERN = /^(-|(<%))?#\s*#{MAGIC_COMMENT_PREFIX}\s*(%>)?/m
```

Comment:
Sorry, I put in `m` when I was still letting the original design put in the comment, which ends in a newline.  You can get rid of it now.